README.md

```diff
@@ -597,10 +640,16 @@ Branch().loadRewardsWithCallback { (changed: Bool, error: NSError!) -> Void in
 }
 ```
 
-### Redeem all or some of the reward balance (store state)
+####Parameters
+
+**callback**: The callback that is called once the request has completed.
+
+### Redeem All or Some of the Reward Balance (Store State)
 
 We will store how many of the rewards have been deployed so that you don't have to track it on your end. In order to save that you gave the credits to the user, you can call redeem. Redemptions will reduce the balance of outstanding credits permanently.
```

Comment:
This description is really confusing to me. Yes, we do keep track of the number of credits you have, but this method specifically is about *spending* those credits. It should probably be re-worded to say as much.